kafkaesk/consumer.py

```diff
@@ -425,25 +434,25 @@ async def healthy(self) -> None:
     # Event handlers
     async def on_partitions_revoked(self, revoked: typing.List[aiokafka.TopicPartition]) -> None:
         if revoked:
+            async with self._processing:
+                if self._auto_commit:
+                    await self._maybe_commit(forced=True)
+
             for tp in revoked:
+                del self._tp[tp]
```

Comment:
should we check if `tp` is in `self._tp` ? this will fail if it's not present

Reply:
true! good catch